ui/app/templates/components/auth-info.hbs

```diff
@@ -10,7 +10,7 @@
           <li class="action">
             <AlertBanner @type="warning" @message="We've stopped auto-renewing your token due to inactivity.
                 It will expire in {{date-from-now auth.tokenExpirationDate interval=1000 hideSuffix=true}}.
-                on {{date-format auth.tokenExpirationDate 'MMMM Do YYYY, h:mm:ss a'}}" />
+                on {{date-format auth.tokenExpirationDate 'MMMM Do yyyy, h:mm:ss a'}}" />
```

Comment:
MMMM is capitalized, but yyyy is not?

Reply:
another excellent question! in the newest version of date-fns`yyyy` (lowercase) represents the calendar year, which is what we want, whereas `YYYY` represents the week numbering year. see the examples here: 

https://github.com/date-fns/date-fns/blob/master/docs/unicodeTokens.md#popular-mistakes